src/ReactiveUI.Tests/MessageBusTest.cs

```diff
@@ -117,18 +110,18 @@ public void RegisteringSecondMessageSourceShouldMergeBothSources()
         }
 
         [Fact]
-        public void MessageBusThreadingTest()
+        public async Task MessageBusThreadingTest()
         {
             var mb = new MessageBus();
             int? listenedThread = null;
             int? otherThread = null;
             int thisThread = Thread.CurrentThread.ManagedThreadId;
 
-            Task.Run(() => {
+            await Task.Run(() => {
```

Comment:
Yeah, calling `Wait` is a big no-no. See [here](https://twitter.com/bradwilson/status/828780630540382208).